guidelines/terms/21/two-dimensional-layout.html

```diff
@@ -0,0 +1,9 @@
+<dt><dfn data-lt="two-dimensional layout">two-dimensional layout</dfn></dt>
+<dd>
+   					
+   <p>layout with a prescribed vertical and horizontal size or aspect ratio
+   </p>
+   <p class="note">See the <a href="https://www.w3.org/TR/pointerevents/#dfn-pointer">Pointer Events
+      definition for "pointer"</a> [[!pointerevents]].</p>
```

Comment:
what the reason for referencing PE here?

Reply:
oops! I think that was the result of a copy from an existing definition I started from. Removed